exercises/python/encryption-context-complete/src/document_bucket/__init__.py

```diff
@@ -0,0 +1,12 @@
+import os
+import sys
+
+import toml
+
+_CONFIG_FILE = os.path.join(sys.prefix, "config/config.toml")
```

Comment:
Should be `os.path.join(sys.prefix, "config", "config.toml")`

Reply:
D'oh, thought I fixed that already. Thanks